README.md

```diff
@@ -1,372 +1,221 @@
-
-```javascript
-      { test: require.resolve('react'), loader: 'expose?React' },
-      
-      // For React 0.14
-      { test: require.resolve('react-dom/server'), loader: 'expose?ReactDOMServer' }, // not in client one, only server
-```
+3. NPM install:
 
+  ```bash
+  npm install
+  ```
 
-#### Sample webpack.server.rails.config.js (ONLY for server rendering)
-Be sure to check out the latest example version of [client/webpack.server.rails.config.js](https://github.com/shakacode/react-webpack-rails-tutorial/blob/master/client/webpack.server.rails.config.js).
-
-```javascript
-  // Common webpack configuration for server bundle
-
-module.exports = {
-
-  // the project dir
-  context: __dirname,
-  entry: ['./app/startup/serverGlobals'],
-  output: {
-    filename: 'server-bundle.js',
-    path: '../app/assets/javascripts/generated',
-
-    // CRITICAL to set libraryTarget: 'this' for enabling Rails to find the exposed modules IF you
-    //   use the "expose" webpackfunctionality. See startup/serverGlobals.jsx.
-    // NOTE: This is NOT necessary if you use the syntax of global.MyComponent = MyComponent syntax.
-    // See http://webpack.github.io/docs/configuration.html#externals for documentation of this option
-    //libraryTarget: 'this',
-  },
-  resolve: {
-    extensions: ['', '.webpack.js', '.web.js', '.js', '.jsx', 'config.js'],
-  },
-  module: {
-    loaders: [
-      {test: /\.jsx?$/, loader: 'babel-loader', exclude: /node_modules/},
-
-      // React is necessary for the client rendering:
-      { test: require.resolve('react'), loader: 'expose?React' },
-      { test: require.resolve('react-dom/server'), loader: 'expose?ReactDOMServer' },
-    ],
-  },
-};
-```
+4. Start your Rails server:
 
-## What Happens?
+  ```bash
+  foreman start -f Procfile.dev
+  ```
 
-Here's what the browser will render with a call to the `react_component` helper.
-![2015-09-28_20-24-35](https://cloud.githubusercontent.com/assets/1118459/10157268/41435186-6624-11e5-9341-6fc4cf35ee90.png)
+5. Visit [localhost:3000/hello_world](http://localhost:3000/hello_world)
 
-## Usage
+## How it Works
+The generator installs your webpack files in the `client` folder. Foreman uses webpack to compile your code and output the bundled results to `app/assets/javascripts/generated`, which are then loaded by sprockets. These generated bundle files have been added to your `.gitignore` for your convenience.
 
-*See section below titled "Try it out"*
+Inside your Rails views, you can now use the `react_component` helper method provided by React on Rails.
 
-### Helper Method
-The main API is a helper:
+### Building the Bundles
+Each time you change your client code, you will need to re-generate the bundles. The included Foreman `Procfile.dev` will take care of this for you by watching your JavaScript code files for changes. Simply run `foreman start -f Procfile.dev`.
```

Comment:
We should have one sentence that rough describes what a bundle is.

Maybe:

A bundle is a file of JavaScript setup by Webpack. Your web page may load one or more of these "bundles". For Rails apps, we typically put all the client side bundles in `application.js` for minification.

@alexfedoseev @robwise?